apps/pie-storybook/types/StoryOptions.ts

```diff
@@ -17,9 +17,11 @@ export type StoryBackgrounds = {
 
 type StorybookControls = {
     exclude?: string[],
-    include?: string[]
+    include?: string[],
 }
 export interface StoryOptions {
```

Comment:
I think we can probably import types directly from Storybook, but I would prefer a separate ticket/PR to do this